sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -14,11 +14,52 @@ static struct snd_soc_acpi_codecs tgl_codecs = {
 	.codecs = {"MX98357A"}
 };
 
+static const u64 rt711_0_adr[] = {
+	0x000010025D071100
+};
+
+static const u64 rt1308_1_1_adr[] = {
+	0x000120025D130800
+};
+
+static const u64 rt1308_1_2_adr[] = {
+	0x000122025D130800
+};
```

Comment:
@bardliao , how about to do it as
````
static const u64 rt1308_1_adr[] = {
       0x000120025D130800,
       0x000122025D130800,
};
````
then only one entry for BIT(1)
````
	{
		.mask = BIT(1),
		.num_adr = ARRAY_SIZE(rt1308_1_adr),
		.adr = rt1308_1_adr,
	},
````

Reply:
agree, can we do this in a follow-up PR?

Reply:
Sure, will do that.